assignments/callbacks.js

```diff
@@ -1,28 +1,41 @@
+//JS - II
 const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 function firstItem(arr, cb) {
   // firstItem passes the first item of the given array to the callback function.
+  cb(arr[0]);
 }
 
 function getLength(arr, cb) {
   // getLength passes the length of the array into the callback.
+  cb(arr.length);
 }
 
 function last(arr, cb) {
   // last passes the last item of the array into the callback.
+  cb(arr[arr.length - 1]);
 }
 
 function sumNums(x, y, cb) {
   // sumNums adds two numbers (x, y) and passes the result to the callback.
+  let sum = x + y;
+  cb(sum);
 }
 
 function multiplyNums(x, y, cb) {
   // multiplyNums multiplies two numbers and passes the result to the callback.
+  let product = x * y;
+  cb(product);
 }
 
 function contains(item, list, cb) {
   // contains checks if an item is present inside of the given array/list.
   // Pass true to the callback if it is, otherwise pass false.
+  if(list.includes(item) === true) {
```

Comment:
This works too:
`if(list.includes(item) === true) cb(true);`
`else cb(false);`

When one-lined, you don't need brackets, and the `else` will pair with the `if` directly above it. I would only do this for very short statements like you have here, though, and some linters don't like you to do it in the way that I'm showing you anyway because it makes it harder to edit the code by adding in more lines. Does that make sense?